roles/run_apb/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
This role is nice :+1:

Reply:
Yeah I was thinking of breaking this one out (after adding some stuff like serviceaccount permissioning etc)